packages/dx-react-scheduler-demos/src/demo-sources/scheduler-editing/material-ui/editing-features.jsx

```diff
@@ -0,0 +1,187 @@
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+import FormGroup from '@material-ui/core/FormGroup';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Typography from '@material-ui/core/FormControl';
+import { withStyles } from '@material-ui/core/styles';
+import { ViewState, EditingState, IntegratedEditing } from '@devexpress/dx-react-scheduler';
+import {
+  Scheduler,
+  WeekView,
+  Appointments,
+  AppointmentForm,
+  AppointmentTooltip,
+  DragDropProvider,
+} from '@devexpress/dx-react-scheduler-material-ui';
+
+import { appointments } from '../../../demo-data/appointments';
+
+const styles = theme => ({
+  container: {
+    margin: theme.spacing(2),
+    padding: theme.spacing(2),
+  },
+  text: theme.typography.h6,
+  formControlLabel: {
+    ...theme.typography.caption,
+    fontSize: '1rem',
+  },
+});
+
+const currentDate = '2018-06-27';
+const editingOptionsList = [
+  { id: 'allowAdding', text: 'Adding' },
+  { id: 'allowDeleting', text: 'Deleting' },
+  { id: 'allowUpdating', text: 'Updating' },
+  { id: 'allowResizing', text: 'Resizing' },
+  { id: 'allowDragging', text: 'Dragging' },
+];
+
+const EditingOptionsSelector = withStyles(styles, 'EditingOptionsSelector')(({
```

Comment:
If you are using a functional component, would be better to use `useStyles` instead of `withStyles`.